sky/jobs/state.py

```diff
@@ -34,10 +35,6 @@ def _get_db_path() -> str:
 
 _DB_PATH = _get_db_path()
```

Comment:
Move this closer to use site?

(Technically, googlestyle defines the ordering of members, funcs, etc.; but we haven't been following that ordering consistently.)